nav2_controller/package.xml

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>dwb_controller</name>
-  <version>0.2.2</version>
-  <description>TODO</description>
+  <name>nav2_controller</name>
+  <version>0.2.3</version>
+  <description>Local planner action interface</description>
   <maintainer email="carl.r.delsey@intel.com">Carl Delsey</maintainer>
-  <license>Apache-2.0</license>
+  <license>Apache License 2.0</license>
```

Comment:
Also revert this, there was some work dont to put all the formats to something an automated tool can parse